src/components/10-atoms/link/ui.test.js

```diff
@@ -357,13 +357,13 @@ test('should correctly update icon and text', async t => {
     .expect(await iconSVGPathElement.getAttribute('d'))
     .eql(downIconSignature);
 
-  await t.expect(await axaLink.textContent).eql('Mehr Filter');
+  await t.expect(axaLink.textContent).eql('Mehr Filter');
```

Comment:
Too many `await` can cause a delay?

Reply:
The new version of Testcafe spit out lots of warnings about using them the wrong way - I fixed all warnings.